src/benchmark/cases/sum_array_double.js

```diff
@@ -0,0 +1,29 @@
+function sumArrayDouble(array) {
+    var sum = 0.1 - 0.1;
+
+    for (var i = 0; i < array.length; i++) {
+        sum += array[i];
+    }
+
+    return sum;
+}
+
+function sumArrayFillDouble(array) {
+    for (var i = 0; i < array.length; i++) {
```

Comment:
This  generates the sequence `[0, 0.5, 0.333, 0.25, 0.2]` . Wouldn't it be better if the numbers are slightly bigger? Maybe something like `[1, 1.5, 2, 2.5, 3, 3.5]` ?

Reply:
I think you read the code wrong because this generates the sequence `[0, 0.5, 1, 1.5, 2, 2.5...]`